apps/api/src/app/topics/use-cases/filter-topics/filter-topics.use-case.ts

```diff
@@ -21,21 +21,19 @@ export class FilterTopicsUseCase {
 
     const query = this.mapFromCommandToEntity(command);
 
-    const totalCount = await this.topicRepository.count(query);
-
     const skipTimes = page <= 0 ? 0 : page;
     const pagination = {
       limit: pageSize,
       skip: skipTimes * pageSize,
     };
 
-    const filteredTopics = await this.topicRepository.filterTopics(query, pagination);
+    const { data, totalCount } = await this.topicRepository.filterTopics(query, pagination, command.subscriberId);
```

Comment:
SubscriberId should be passed in the query object. Adding it as an extra param complicates things a bit in a way I find unnecessary when we are already passing an object with the information for the DAL query.